adafruit_ble_midi.py

```diff
@@ -19,6 +19,11 @@
 from adafruit_ble.uuid import VendorUUID
 from adafruit_ble.services import Service
 
+try:
+    from circuitpython_typing import WriteableBuffer, ReadableBuffer
```

Comment:
Can you add a guard typing import of `typing` as the first line in the try block here?  So:

```python
try:
    import typing  # pylint: disable=unused-import
    ...
```